lib/services/navigation_service.dart

```diff
@@ -62,6 +64,78 @@ class NavigationService {
     );
   }
 
+  void showTalawaErrorWidget(
+    String message, {
+    Duration duration = const Duration(seconds: 2),
+  }) {
+    ScaffoldMessenger.of(navigatorKey.currentContext!).showSnackBar(SnackBar(
+      padding: EdgeInsets.zero,
```

Comment:
Could you refactor this into a reusable stateless widget that takes message as the parameter?
That way we can ensure snackbars are consistent across the app

Reply:
You also probably want to change to a auto sized text as well, incase the message is long or complex

Reply:
I made the changes you asked for, please  check the PR